client/me/purchases/manage-purchase/index.jsx

```diff
@@ -654,7 +655,9 @@ class ManagePurchase extends Component {
 				{ siteId && <QuerySiteDomains siteId={ siteId } /> }
 				{ isPurchaseTheme && <QueryCanonicalTheme siteId={ siteId } themeId={ purchase.meta } /> }
 
-				<FormattedHeader brandFont headerText={ titles.sectionTitle } align="left" />
+				{ showHeader && (
```

Comment:
Instead of a prop, would it be better to declare the header inside `<Main>` where the component is used.